### DISCUSSION TITLE
term.xplr - Generic terminal support for xplr

### DISCUSSION THREAD

**Discussion by igorepst** [sayanarijit/xplr#386] [Show and tell]
Hi,
I developed a generic plugin to support terminals via xplr, based on the [alacritty.xplr](https://github.com/sayanarijit/alacritty.xplr).
Please see [term.xplr](https://github.com/igorepst/term.xplr).
TODO:
Support profiles (almost zero config) to use [kitty](https://github.com/kovidgoyal/kitty) splits/tabs. The splits or tabs may be used right now, but for that you need to follow example from the README to define arguments

**Comment by sayanarijit** [comment]
To add it in the docs, you can create a pr updating this file: https://github.com/sayanarijit/xplr/blob/main/docs/en/src/awesome-plugins.md

**Comment by igorepst** [comment]
@sayanarijit  Thank you for adding my plugin! I didn't want to announce it before I find a time to support multiple profiles. I pushed the support today (e.g. now you may press 'Ctrl+N' to open `xplr` in vertical split of `kitty`, and some other key - to open in `alacritty`) and intended to open a PR for you to announce the plugin, but suddenly saw it was announced already :)

**Reply by sayanarijit** [reply]
FYI: I pushed some changed here: https://github.com/sayanarijit/alacritty.xplr/pull/4 which fixes some bugs.

**Reply by sayanarijit** [reply]
Also: https://github.com/sayanarijit/alacritty.xplr/pull/5

**Comment by igorepst** [comment]
Merged, tnx!
